ansible/deploy.yml

```diff
@@ -239,15 +239,24 @@
           port: "{{ node_config.http.external.port }}"
           host: "{{ public_ipv4 }}"
           timeout: 300
-        when: health_check
+        when: health_check and not api_gateway
         connection: local
         tags: [health-check]
 
       - name: API health check
         uri:
           url: "{{ api_base_uri }}/blocks/top"
           timeout: 30
-        when: health_check
+        when: health_check and not api_gateway
```

Comment:
what if you set `health_check: false` for `api-main` instead of this additional check?

Reply:
that was my orginal idea, but you told me to have additional variable.

Reply:
I guess it was misunderstanding  👍